<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Bilge-Katalog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtRc.qrc">
    <normaloff>:/image/images/bilge-katalog.png</normaloff>:/image/images/bilge-katalog.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QStackedWidget" name="viewCat">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <widget class="QWidget" name="pageListCat">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QListWidget" name="listCat"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageTreeCat">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTreeWidget" name="treeCat">
           <property name="columnCount">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="viewFiles">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <widget class="QWidget" name="pageListFiles">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QListWidget" name="listFiles"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageTableFiles">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QListWidget" name="tableFiles"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageIconFile">
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QListWidget" name="iconFiles">
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::IconMode</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actCreateCatalog"/>
    <addaction name="separator"/>
    <addaction name="actExpCat"/>
    <addaction name="actImpCat"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="actNewDir"/>
     <addaction name="actNewFile"/>
    </widget>
    <widget class="QMenu" name="menuUsers">
     <property name="title">
      <string>&amp;Users</string>
     </property>
     <addaction name="actNewUser"/>
     <addaction name="actUserInfo"/>
    </widget>
    <addaction name="menuUsers"/>
    <addaction name="menuNew"/>
    <addaction name="actDel"/>
    <addaction name="actCut"/>
    <addaction name="actCopy"/>
    <addaction name="actPaste"/>
    <addaction name="actRefresh"/>
    <addaction name="actLending"/>
    <addaction name="actInfo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuCatView">
     <property name="title">
      <string>&amp;Catalog view</string>
     </property>
     <addaction name="actCatListView"/>
     <addaction name="actCatTreeView"/>
    </widget>
    <widget class="QMenu" name="menuFilesView">
     <property name="title">
      <string>Files v&amp;iew</string>
     </property>
     <addaction name="actFileListView"/>
     <addaction name="actFileTableView"/>
     <addaction name="actFileIconView"/>
    </widget>
    <addaction name="actShowEditTB"/>
    <addaction name="actShowExploreTB"/>
    <addaction name="actShowSearchTB"/>
    <addaction name="actShowStat"/>
    <addaction name="separator"/>
    <addaction name="menuCatView"/>
    <addaction name="menuFilesView"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="exploreToolBar">
   <property name="windowTitle">
    <string>Explore Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>Edit Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="searchToolBar">
   <property name="windowTitle">
    <string>Search Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actExpCat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Export Catalog</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actImpCat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Import Catalog</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/exit.png</normaloff>:/image/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actShowStat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;status bar</string>
   </property>
  </action>
  <action name="actShowEditTB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show edi&amp;t toolbar</string>
   </property>
  </action>
  <action name="actCatTreeView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Tree</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actCatListView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;List</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actFileTableView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Table</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actFileListView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;List</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actDel">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/delete.png</normaloff>:/image/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actFileIconView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ic&amp;ons</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actCreateCatalog">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/catalog_new.png</normaloff>:/image/images/catalog_new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Create Catalog</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actNewDir">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/directory_new.png</normaloff>:/image/images/directory_new.png</iconset>
   </property>
   <property name="text">
    <string>Dire&amp;ctory</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actNewFile">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/file_new.png</normaloff>:/image/images/file_new.png</iconset>
   </property>
   <property name="text">
    <string>Fi&amp;le</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actShowExploreTB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show e&amp;xplore toolbar</string>
   </property>
  </action>
  <action name="actShowSearchTB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show sea&amp;rch toolbar</string>
   </property>
  </action>
  <action name="actBack">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/back.png</normaloff>:/image/images/back.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actNext">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/next.png</normaloff>:/image/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actUp">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/up.png</normaloff>:/image/images/up.png</iconset>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/copy.png</normaloff>:/image/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actCut">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/cut.png</normaloff>:/image/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/paste.png</normaloff>:/image/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actRefresh">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/refresh.png</normaloff>:/image/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actInfo">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/info.png</normaloff>:/image/images/info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Info</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actLending">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/lend.png</normaloff>:/image/images/lend.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Lending</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actNewUser">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/user_new.png</normaloff>:/image/images/user_new.png</iconset>
   </property>
   <property name="text">
    <string>Ne&amp;w User</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actUserInfo">
   <property name="icon">
    <iconset resource="QtRc.qrc">
     <normaloff>:/image/images/user.png</normaloff>:/image/images/user.png</iconset>
   </property>
   <property name="text">
    <string>U&amp;ser Info</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtRc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actShowExploreTB</sender>
   <signal>triggered(bool)</signal>
   <receiver>exploreToolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actShowEditTB</sender>
   <signal>triggered(bool)</signal>
   <receiver>editToolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actShowStat</sender>
   <signal>triggered(bool)</signal>
   <receiver>statusBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actShowSearchTB</sender>
   <signal>triggered(bool)</signal>
   <receiver>searchToolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
